pkg/ingester/flush.go

```diff
@@ -139,11 +139,9 @@ func (i *Ingester) flushLoop(j int) {
 		}
 		op := o.(*flushOp)
 
-		level.Debug(util_log.Logger).Log("msg", "flushing stream", "userid", op.userID, "fp", op.fp, "immediate", op.immediate)
-
 		err := i.flushUserSeries(op.userID, op.fp, op.immediate)
 		if err != nil {
-			level.Error(util_log.WithUserID(op.userID, util_log.Logger)).Log("msg", "failed to flush user", "err", err)
+			level.Error(util_log.WithUserID(op.userID, util_log.Logger)).Log("msg", "failed to flush", "err", err, "user", op.userID)
```

Comment:
I think I would leave this alone for now, this is a pretty well established error message.

Also, the logger includes the user ID so we shouldn't have to add it at the end.

Reply:
Good point, done!

Reply:
Also after some thought I'm fine with removing `user` from that error message.

Reply:
> Good point, done!

Not sure if the push didn't make it or if github is playing tricks on me, but I still see `user` in the list of args?

Reply:
Nope, I'd just missed the one case you actually pointed out but did the other others :upside_down_face: